tests/platform_tests/api/test_sfp.py

```diff
@@ -181,6 +181,14 @@ def test_get_presence(self, duthosts, enum_rand_one_per_hwsku_hostname, localhos
                     self.expect(presence is True, "Transceiver {} is not present".format(i))
         self.assert_expectations()
 
+    def test_get_error_description(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platform_api_conn):
+        for i in self.candidate_sfp:
+            error_description = sfp.get_error_description(platform_api_conn, i)
+            if self.expect(error_description is not None, "Unable to retrieve transceiver {} error description".format(i)):
+                if self.expect(isinstance(error_description, str) or isinstance(error_description, unicode), "Transceiver {} error description appears incorrect".format(i)):
+                    self.expect(error_description == "OK", "Transceiver {} is not present".format(i))
+        self.assert_expectations()
```

Comment:
Why add this under the `Functions to test methods inherited from DeviceBase class` section? `get_error_description` is defined in SfpBase. Please move under `Functions to test methods defined in SfpBase class` section.

Reply:
Fixed.